aserto/directory/model/v3/model.ext.openapi.json

```diff
@@ -39,6 +39,28 @@
                   "format": "binary"
                 }
               }
+            },
+            "headers": {
+              "X-Created-At": {
+                "description": "manifest creation time",
+                "schema": {
+                  "type": "string",
+                  "format": "date-time"
+                }
+              },
+              "X-Updated-At": {
```

Comment:
I would have picked updated_at as Last-Modified, but I agree that since we will not support If-Modified-Since we should not do this, I would simply name them like our other headers
aserto-created-at
aserto-updated-at
aserto-hash or aserto-etag